packages/components/psammead-social-embed/src/__snapshots__/index.test.jsx.snap

```diff
@@ -2,6 +2,54 @@
 
 exports[`AmpSocialEmbed should render a notice when the provider is unsupported 1`] = `
 .c0 {
+  position: relative;
+}
+
+.c1 {
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 0.875rem;
+  line-height: 1rem;
+  background-color: #FFFFFF;
+  border: 0.125rem solid #222222;
+  color: #222222;
+  display: block;
+  left: 0;
+  line-height: 1;
+  padding: 0.75rem;
+  position: absolute;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  top: 0;
+  z-index: 10;
```

Comment:
Is this a random number? why 10?

Reply:
z-index values tend to be arbitrary. Here we're ensuring the skip link sits above the oEmbed.

Reply:
Ok, was just checking... thanks